.github/workflows/test_with_drupal.yaml

```diff
@@ -0,0 +1,37 @@
+name: Test With Drupal
+
+on:
+    pull_request: null
+    push:
+        branches:
+            - master
+
+jobs:
+    test_with_drupal:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v1
+                with:
+                    php-version: 7.3
+                    coverage: none
+
+            -   name: Install Drupal project
+                run: |
+                    # bring rector dependencies
+                    composer install --no-progress --no-dev
```

Comment:
The `--no-dev` needs to be removed, otherwise local `phpstan.neon` configs are missing
https://github.com/rectorphp/rector/pull/2891/checks?check_run_id=456122559#step:5:4